.github/workflows/ci.yml

```diff
@@ -42,9 +42,8 @@ jobs:
       #        if: ${{ steps.cache-node-modules.outputs.cache-hit != 'true' }}
 
       - name: Install packages
-        if: ${{ steps.cache-node-modules.outputs.cache-hit != 'true' }}
```

Comment:
I need to add this line of code back after we merge this, wanted to just trigger the yarn installs while I was still making updates

Reply:
If the lock file is part of the cache key, how are we getting hits? I would assume that the yarn lock file would be slightly different? Another option would be to slightly tweak the cache key to something like: `${{ runner.os }}-${{ env.cache-name }}-yarn-${{ hashFiles('**/yarn.lock') }}`.